lib/pdn/esm.c

```diff
@@ -49,7 +49,6 @@ static const struct {
 	{ 0x3a, "PDN type non IP only allowed" },
 	{ 0x3b, "Unsupported QCI value" },
 	{ 0x3c, "Bearer handling not supported" },
-	{ 0x3d, "PDN type Ethernet only allowed" },
```

Comment:
So 3GPP has removed this cause value?

Reply:
Yes, it seems so, that cause is not present in the rel 14 spec. I couldn't find any other changes.